robottelo/hosts.py

```diff
@@ -257,6 +257,14 @@ def os_version(self):
         version_string = f'{groups["major"]}{minor_version}'
         return Version(version=version_string)
 
+    @cached_property
+    def is_fips_enabled(self):
+        """Check if the host is FIPS enabled.
+
+        :returns: A ``bool`` indicating whether FIPS is enabled or not.
+        """
+        return 'crypto.fips_enabled = 1' in self.execute('sysctl crypto.fips_enabled').stdout
```

Comment:
I think this will fail with container content host as sysctl might not be available there. If yes, we should also add a condition to check only on non container content host.

Reply:
removing for now as it is not been used